medusa/show_updater.py

```diff
@@ -178,6 +186,36 @@ def expired_seasons(self):
 
         return seasons_dict
 
+    def clean_expired_seasons(self, show_list):
```

Comment:
I suspect this is only required to cleanup the mess I made. Maybe we can also just remove our cache.db. And live without the rather (complex) method.